### ISSUE TITLE
Not possible to debug external Python code

### ISSUE THREAD

**Issue by PiotrNestor** [getgauge/gauge-python#265]
**Gauge python in VScode : Unable to debug external code**

**Describe the solution you'd like**
Provide configuration support for
"justMyCode": false

It should be possible to debug external Python packages while in debug mode

**Describe alternatives you've considered**

**Additional context**
Windows 10 / 11 Pro
Latest VSCode version

**Comment by nagamanickamm** [comment]
Yes, I too face the same issue. But looks like there are no contributors to this library anymore.
I'm stuck choosing gauge python
